src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract String doWork();

```

Comment:
![image](https://user-images.githubusercontent.com/12864128/200576080-7836b08d-d550-4a49-a62c-e3711438d816.png)
Method should return nothing.